tests/FlatBuffers.Test.Nim/NimTest.sh

```diff
@@ -0,0 +1,19 @@
+nim_dir=`pwd`
```

Comment:
I prefer having these scripts written in python, since then it can work on windows, mac, and linux. 

See: https://github.com/google/flatbuffers/blob/master/tests/ts/TypeScriptTest.py

Or better yet: https://github.com/google/flatbuffers/blob/master/scripts/generate_grpc_examples.py since that one uses a common `util.py` for defining flatc python method.